src/panels/lovelace/cards/hui-light-card.ts

```diff
@@ -99,7 +99,7 @@ export class HuiLightCard extends LitElement implements LovelaceCard {
     }
 
     const brightness =
-      Math.round((stateObj.attributes.brightness / 254) * 100) || 0;
+      Math.round((stateObj.attributes.brightness / 255) * 100) || 0;
```

Comment:
Should this be 1 now?
```suggestion
      Math.round((stateObj.attributes.brightness / 255) * 100) || 1;
```

Reply:
I am not sure, but I think no. If for some reason we get invalid brightness value so that the first part results in an NaN, then we should not switch on the light slightly dimmed at 1%, but it should stay at 0. Or am I missing something here?